redis/src/types.rs

```diff
@@ -759,6 +770,14 @@ impl RedisWrite for Vec<Vec<u8>> {
     fn write_arg_fmt(&mut self, arg: impl fmt::Display) {
         self.push(arg.to_string().into_bytes())
     }
+
+    fn start_buffered_arg(&mut self) {
+        self.push(Vec::new());
+    }
+    fn write_buffered_arg(&mut self, arg: &[u8]) {
+        self.last_mut().unwrap().extend_from_slice(arg);
```

Comment:
Should we return a `Result<...>` of some type if `last_mut()` is `None`?

Reply:
Right now I think I'm ok with having it panic because:
1. `ToRedisArgs::to_redis_args`, which would be the main caller of this method, doesn't itself return a `Result` (this is also why `RedisWrite::write_arg` doesn't return a `Result`), so there really wouldn't be any option to handle the error other than panicking one level up anyways
2. afaict the `Vec<Vec<u8>>` implementation is only used in test-type environments (this code isn't exercised at all actually, and is just here for completion)